.azuredevops/components/roctracer.yml

```diff
@@ -56,9 +62,15 @@ jobs:
     parameters:
       extraBuildFlags: >-
         -DCMAKE_BUILD_TYPE=release
-        -DCMAKE_MODULE_PATH="$(Agent.BuildDirectory)/rocm/lib/cmake"
-        -DCMAKE_PREFIX_PATH="$(Agent.BuildDirectory)/rocm/llvm;$(Agent.BuildDirectory)/rocm"
-        -DENABLE_LDCONFIG=OFF
+        -DROCM_PATH=$(Agent.BuildDirectory)/rocm
+        -DCMAKE_MODULE_PATH=$(Agent.BuildDirectory)/rocm/lib/cmake/hip
+        -DCMAKE_PREFIX_PATH=$(Agent.BuildDirectory)/rocm
         -DGPU_TARGETS=gfx1030;gfx1100
         -GNinja
-  - template: ${{ variables.CI_TEMPLATE_PATH }}/steps/artifact-upload.yml
+#  - template: ${{ variables.CI_TEMPLATE_PATH }}/steps/artifact-upload.yml
+#  - task: Bash@3
+#    displayName: 'Tests'
+#    inputs:
```

Comment:
Shall we delete comments for the dev branch?

Reply:
It's okay to leave it there for reference for when we add tests. The commands work but need to be run on a system with gpu.